### ISSUE TITLE
Bomber should show the bomber picture, when flying

### ISSUE THREAD

**Issue by pavelsuk** [simwir/BomberTutorial#3]
It's currently using car image.

**Comment by simwir** [comment]
To make the mod work with the latest version i just copied the code for displaying the car over to the bomber, to get a working version. The code on the wiki does not appy there anymore. If you can tweak the code to show the plane instead please go ahead :smiley:

**Comment by pavelsuk** [comment]
Yeah, I realized that. The tutorial is great, but very outdated. Thanks for pushing fixed version to github, it saved me a lot of time.

**Comment by simwir** [comment]
You are very welcome, thank you for helping keeping it as correct and current as possible
